### ISSUE TITLE
evaluate-quantized step fails in CI

### ISSUE THREAD

**Issue by eu9ene** [mozilla/translations#454]
https://firefox-ci-tc.services.mozilla.com/tasks/GAul9abYQ5amzWKxb9i64Q/runs/0/logs/public/logs/live.log

It's likely an issue introduced after refactoring. The `quantize` step often fails in CI so we didn't see this before.

**Comment by gregtatum** [comment]
The log expired and is no longer there. I don't recall this happening recently so I'll close it.

**Comment by ZJaume** [comment]
This is probably failing because it is using binary shortlist parameter with a shortlist in text format
```
... --shortlist /builds/worker/fetches/lex.s2t.pruned false ...
```
and will be fixed by #1169
